Examples/Swift/ManageOfflineRegionsExample.swift

```diff
@@ -0,0 +1,197 @@
+
+    lazy var mapView: MGLMapView = {
+        let mapView = MGLMapView(frame: CGRect.zero)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.tintColor = .gray
+        mapView.delegate = self
+        mapView.translatesAutoresizingMaskIntoConstraints = false
+        return mapView
+    }()
+
+    lazy var downloadButton: UIButton = {
+        let downloadButton = UIButton(frame: CGRect.zero)
+        downloadButton.backgroundColor = UIColor.white
+        downloadButton.setTitleColor(UIColor.systemBlue, for: .normal)
+        downloadButton.setTitle("Download Region", for: .normal)
+        downloadButton.addTarget(self, action: #selector(startOfflinePackDownload), for: .touchUpInside)
+        downloadButton.layer.cornerRadius = view.bounds.width / 30
+        downloadButton.translatesAutoresizingMaskIntoConstraints = false
+        return downloadButton
+    }()
+
+    lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: CGRect.zero)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+
+    override func viewDidLoad() {
+
+        super.viewDidLoad()
+
+        view.addSubview(mapView)
+        view.addSubview(tableView)
+        mapView.addSubview(downloadButton)
+        let centerCoordinate = CLLocationCoordinate2D(latitude: 22.27933, longitude: 114.16281)
+        mapView.setCenter(centerCoordinate, zoomLevel: 13, animated: false)
+
+        // Setup offline pack notification handlers.
+        setupOfflinePackHandler()
+        installConstraints()
+
+    }
+
+    func setupOfflinePackHandler() {
+        NotificationCenter.default.addObserver(self, selector: #selector(offlinePackProgressDidChange), name: NSNotification.Name.MGLOfflinePackProgressChanged, object: nil)
+    }
+
+    func installConstraints() {
+
+        NSLayoutConstraint.activate([
+            mapView.topAnchor.constraint(equalTo: view.topAnchor),
+            mapView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            mapView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            mapView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 0.75),
+            tableView.topAnchor.constraint(equalTo: mapView.bottomAnchor),
+            tableView.leadingAnchor.constraint(equalTo: mapView.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: mapView.trailingAnchor),
+            tableView.heightAnchor.constraint(equalTo: mapView.heightAnchor, multiplier: 0.4),
+            downloadButton.topAnchor.constraint(equalTo: mapView.topAnchor, constant: 100),
+            downloadButton.leadingAnchor.constraint(equalTo: mapView.leadingAnchor, constant: 5),
+            downloadButton.trailingAnchor.constraint(equalTo: mapView.leadingAnchor, constant: 150)
```

Comment:
I think there are some improvements that can be made here -

**Map view / table view**:

The only constraints here that should be relative to the `mapView` are the table view's top. The rest can just be relative to the `view` itself.

<details>
<summary>OR, you can take on a ✨ bonus challenge ✨... </summary>

<br>

The map view and table view take up almost 50/50 in height. Instead of manually setting the constrains for these views, I challenge you to experiment with placing both of these in a [`UIStackView`](https://developer.apple.com/documentation/uikit/uistackview). The `UIStackView` will need to have its `distribution` and `axis` properties set to the correct values for this to work.
</details>

**Download button**:

Instead of doing a trailing anchor for the download button, let's set a `width` constraint instead. I'd also add some `height` since the button looks a little squished:

![image](https://user-images.githubusercontent.com/10850812/99131895-3acb7a80-25c9-11eb-8ac4-2a589f125ae8.png)


To fit with the design and to stand out from the map more, can you reverse the button's text color and background color?